### DISCUSSION TITLE
On mount console.log is executed two times in `@codesandbox/sandpack-react`

### DISCUSSION THREAD

**Discussion by VladSez** [codesandbox/sandpack#748] [Q&A]
On initial mount console.log is executed two times (check logs at bottom right):

![image](https://user-images.githubusercontent.com/16666234/220343812-7b16d80d-3820-41fb-8e68-efc97e7b54c2.png)

Reproduction:
https://codesandbox.io/s/sweet-roentgen-v2kdl6?file=/src/App.js

Is it expected? Can I make it to execute only one time on mount?

PS: sometimes `console.log` is executed only once but most of the time it's executed twice

**Comment by andysm1th** [comment]
This is probably because you're running React in strict mode.
